config/config.ts

```diff
@@ -125,21 +125,26 @@ export default {
               path: '/ssl',
               icon: 'crown',
               routes: [
+                {
+                  path: '/ssl',
+                  redirect: '/ssl/list',
+                },
                 {
                   path: '/ssl/list',
                   name: 'list',
-                  component: './SSLModule/list',
```

Comment:
Please keep using `XXXModule`.

Reply:
We used to organize documents like this
> ![image](https://user-images.githubusercontent.com/8078418/79034442-77289980-7be8-11ea-8da8-8a0bee616234.png)

I think it's more helpful to differentiate functional modules by changing it to the following form.
> ![image](https://user-images.githubusercontent.com/8078418/79034453-81e32e80-7be8-11ea-93d2-0270953aeed4.png)

This requires discussion to determine the best option.

Reply:
`module` is respected as `reversed keyword`, we'd better not use it 😂Also, most features are grouped as `XXXModule`, so please keep using it in my opinion :)

Reply:
If so, i think it might be better to use XXXModule.

Reply:
Yep, we use `FeatureGroupModule` style.

Reply:
Ok, changing in process.